specs/_features/maxeb_increase/capella.py

```diff
@@ -3698,20 +3714,32 @@ def get_expected_withdrawals(state: BeaconState) -> Sequence[Withdrawal]:
     validator_index = state.next_withdrawal_validator_index
     withdrawals: List[Withdrawal] = []
     for i, withdrawal in enumerate(state.pending_balance_withdrawals):
-        if validator.slashed:
-           withdrawal.withdrawable_epoch = validator.withdrawable_epoch 
-        if withdrawal.withdrawable_epoch <= epoch:
+        if withdrawal.withdrawable_epoch > epoch:
```

Comment:
we can only break if we know that withdrawal.withdrawable_epoch is monotone right? do we have that certainty?